### PR TITLE
create vmnet8.md

### PR DESCRIPTION
**Pull request by vmnet8**
Added self introduction, please review.

### CODE REVIEW

**Comment by dogi** [comment]
@vmnet8 generally it look ok

the raw githack link for testing is missing here in the description

**Comment by vmnet8** [comment]
> @vmnet8 generally it look ok
> 
> the raw githack link for testing is missing here in the description

https://raw.githack.com/vmnet8-OLE/vmnet8-OLE.github.io/treehouses_migration/#!pages/vi/profiles/vmnet8.md

**Review by Liew211 — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Review by lillyxxx — APPROVED** [review]

File: pages/vi/profiles/vmnet8.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+* I like penguins.
+
+### penguin image
+
+![penguin](https://media.mnn.com/assets/images/2019/02/emperor-penguin-juvenile-adults-book.jpeg.653x0_q80_crop-smart.jpg)
+
+
+You can find my blog here
+[My tech blog](https://vmnet8.github.io/)
```
Comment (dogi) [inline pages/vi/profiles/vmnet8.md:17]: @vmnet8 awesome blog

**Review by dogi — APPROVED** [review]

File: pages/vi/profiles/vmnet8.md (hunk 1 above)
Comment (vmnet8) [reply pages/vi/profiles/vmnet8.md:17]: Thank you!

**Pull request merged**